Code-Hammers/code-hammers
# Che 127/add remaining profile info to profile page

### Description

As part of [CHE-113](https://codehammers.atlassian.net/browse/CHE-113) story "profiles improvement", this PR adds the remaining profile information to the profile page. 

### Jira Task

[CHE-127](https://codehammers.atlassian.net/jira/software/projects/CHE/list?direction=ASC&sortBy=labels&selectedIssue=CHE-127)

### Testing Instructions

1. Make sure all branches are up to date and you are working from feature branch
2. Spin up the dev environment and log in
3. Select "Profiles" from the nav bar
4. Click on a profile and ensure all information is being displayed appropriately 

NOTE: social media info (clickable icon) will only be displayed when a valid link is given. If not displayed, navigate to the edit profiles page and insert valid social media links for the profile being tested (see instructions below). 

#### Instructions to add social media links 

1. Click on "account" then "edit profile" in the nav bar
2. Scroll down to the various social media inputs
3. Enter valid links for each site and click "save changes" when done
4. Follow instructions above to navigate back to the profile page to verify that the social media icons are now present


### Checklist

- [X] I ran `npm run docker-test:all` in my local environment to check that this PR passes all unit tests.


[CHE-113]: https://codehammers.atlassian.net/browse/CHE-113?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[CHE-127]: https://codehammers.atlassian.net/browse/CHE-127?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
